language/tools/cost_synthesis/src/vm_runner.rs

```diff
@@ -43,6 +43,10 @@ macro_rules! with_loaded_vm {
         let mut $vm =
             TransactionExecutor::new(&$module_cache, &data_cache, TransactionMetadata::default());
         $vm.turn_off_gas_metering();
-        $vm.execution_stack.push_frame(entry_func);
+        match $vm.execution_stack.push_frame(entry_func) {
+            Ok(Ok(_)) => {}
+            Ok(Err(e)) => panic!("Unexpected Runtime Error: {:?}", e),
+            Err(e) => panic!("Unexpected ICE: {:?}", e),
```

Comment:
Offhand, I can't recall what ICE means. Why not just spell it out?

Reply:
This is the current standard format for handing such errors so I am conforming to the local style. See the `assert_ok` macro in `vm/src/errors.rs`.